eva-accession-pipeline/src/test/java/uk/ac/ebi/eva/accession/pipeline/io/AccessionWriterTest.java

```diff
@@ -127,16 +131,20 @@ private void assertVariantEquals(ISubmittedVariant expectedvariant, ISubmittedVa
         assertEquals(expectedvariant.getStart(), actualVariant.getStart());
         assertEquals(expectedvariant.getReferenceAllele(), actualVariant.getReferenceAllele());
         assertEquals(expectedvariant.getAlternateAllele(), actualVariant.getAlternateAllele());
+        assertEquals(expectedvariant.getClusteredVariant(), actualVariant.getClusteredVariant());
         assertEquals(expectedvariant.isSupportedByEvidence(), actualVariant.isSupportedByEvidence());
+        assertEquals(expectedvariant.getMatchesAssembly(), actualVariant.getMatchesAssembly());
     }
```

Comment:
Same comment regarding the `equals` method as in `ImportedSubmittedVariantWriterTest.java`.